www-src/app/lib/translation.coffee

```diff
@@ -0,0 +1,23 @@
+Polyglot = require 'node-polyglot'
+
+module.exports = class Translation
+    DEFAULT_LANGUAGE: 'en'
+
+    constructor: ->
+        @polyglot = new Polyglot()
+
+    initialize: (locale) ->
+        [@language] = locale.value.split '-'
+        @_getTranslationsFromFile()
+
+        @polyglot.t.bind @polyglot
```

Comment:
@kosssi OK, I was lost because it seems weird to assign the return of something called `initialize`. You should probably move this to a distinct helper.

And maybe rename `initialize` in `setLocale`.